client/lib/fse-beta/send-site-editor-beta-feedback.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import config from '@automattic/calypso-config';
+/**
+ * Internal dependencies
+ */
+import wpcom from 'calypso/lib/wp';
+
+export const sendSiteEditorBetaFeedback = ( message, siteUrl, userLocale, port ) => {
+	const kayakoMessage = `Site: ${ siteUrl ? siteUrl : 'N/A' }\n\n${ message }`;
+
+	// To test without sending.
+	// console.log( 'test message:' );
+	// console.log( kayakoMessage );
+	// port.postMessage( 'success' );
+	// return;
```

Comment:
To remove - just for simple testing while developing without zendesk access or spamming tickets.